### PR TITLE
Update Electron + Continuous Progress + Wiki Link

### PR DESCRIPTION
**Pull request by flavioislima**
- [x] Update Electron to v12
- [x] Add continuous progress feature to install and updates
- [x] Add Wiki Link

Updated version to 1.7 since next version will be a feature release.

Header Change:
image

### CODE REVIEW

File: src/components/utils.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { InstallProgress } from './../types';
```
Comment (adityaruplaha) [inline src/components/utils.ts:1]:
```suggestion
import { InstallProgress } from '../types';
```

File: src/components/utils.ts [hunk 2]
```diff
@@ -43,7 +46,7 @@ export async function handleInstall({
   if (installPath === 'import') {
     const { filePaths } = await showOpenDialog({
       buttonLabel: t('gamepage:box.choose'),
-      properties: ['gamepage:openDirectory'],
+      properties: ['openDirectory'],
```
Comment (adityaruplaha) [inline src/components/utils.ts:49]: What happened here?

File: src/components/Navbar/index.css [hunk 3]
```diff
@@ -7,7 +7,7 @@
   box-shadow: 0px 4.00313px 10.0078px #161616;
   width: 100%;
   display: grid;
-  grid-template-columns: 3fr 5fr 1fr;
+  grid-template-columns: 5fr 5fr 1fr;
```
Comment (adityaruplaha) [inline src/components/Navbar/index.css:10]: A screenshot would be helpful in visualising the difference. Minor thing though.

**Review by adityaruplaha — COMMENTED** [review]
Did a general look through on mobile. Looks good mostly. Couple of things I would like explanation on.

File: src/components/utils.ts (hunk 1 above)
Comment (flavioislima) [reply src/components/utils.ts:1]: Weird that the automatic import got the wrong file? 🤔

File: src/components/utils.ts (hunk 2 above)
Comment (flavioislima) [reply src/components/utils.ts:49]: This is a bug. If you look at it, we should have a property but we are using a translation key. I fixed this long time ago but I think got back on some rebase.
This bug make it impossible to select a folder when installing from the library.

File: src/components/Navbar/index.css (hunk 3 above)
Comment (flavioislima) [reply src/components/Navbar/index.css:10]: Ok. But this basically increased the size of the navbar to accommodate the wiki button. Without it, in some languages the wiki button was colliding with the search bar.

File: src/components/utils.ts (hunk 1 above)
Comment (flavioislima) [reply src/components/utils.ts]: Fixed

File: src/components/Navbar/index.css (hunk 3 above)
Comment (flavioislima) [reply src/components/Navbar/index.css:10]: Just added

**Comment by flavioislima** [comment]
@adityaruplaha ignore the i18n changed files, thats because weblate uses 4 indentation spaces on JSON and there is no option to change that, and we were using 2 spaces, so I changed the config and generated the files again to avoid conflicts with it.

File: src/components/Navbar/index.css (hunk 3 above)
Comment (adityaruplaha) [reply src/components/Navbar/index.css:10]: > Ok. But this basically increased the size of the navbar to accommodate the wiki button. Without it, in some languages the wiki button was colliding with the search bar. 

Alright. Looks good then.

**Review by adityaruplaha — APPROVED** [review]
LGTM.

**Pull request merged**